src/components/GlassButton.vue

```diff
@@ -42,55 +38,56 @@
 </template>
 
 <script setup lang="ts">
+import { Icon, IconifyIcon } from '@iconify/vue'
 import { computed } from 'vue'
 
 const props = defineProps<{
   /**
-   * The text label displayed on the button (rectangular variant) or under the button (round).
+   *
    */
   label?: string
   /**
-   * Additional Tailwind classes for the label.
+   *
    */
   labelClass?: string
   /**
-   * The icon to display in the button (prepend on rectangular, centered on round).
+   *
    */
-  icon?: string
+  icon?: string | IconifyIcon
   /**
-   * Additional Tailwind classes for the icon.
+   *
    */
   iconClass?: string
   /**
-   * The size of the icon.
+   *
    */
   iconSize?: number
   /**
-   * Additional Tailwind classes for the button.
+   *
    */
   buttonClass?: string
   /**
-   * Whether the button is disabled.
+   *
    */
   disabled?: boolean
   /**
-   * The tooltip text displayed on hover.
+   *
    */
   tooltip?: string
   /**
-   * Visual feedback for the button when toggled selected.
+   *
    */
   selected?: boolean
   /**
-   * The shape of the button.
+   *
    */
   variant?: 'round' | 'rectangular' | 'uncontained'
   /**
-   * The width of the button (when round, this is the only size nedded).
+   *
    */
   width?: number
   /**
-   * The height of the button.
+   *
```

Comment:
Why were the docs removed here?

Reply:
Yes it was a mistake. Putting them back...